### PR TITLE
example/pidof.py same as pidof cli tool mentioned in TODO:

### PR DESCRIPTION
**Pull request by karthikrev**

### CODE REVIEW

File: examples/pidof.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
```
Comment (giampaolo) [inline examples/pidof.py]: Add the license here:

# Copyright (c) 2009, Giampaolo Rodola'. All rights reserved.
# Use of this source code is governed by a BSD-style license that can be
# found in the LICENSE file.

File: examples/pidof.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+"""
+A clone of 'pidof' cmdline utility.
+$ pidof  /usr/bin/python
+1140 1138 1136 1134 1133 1129 1127 1125 1121 1120 1119
+"""
+
```
Comment (giampaolo) [inline examples/pidof.py]: from __future__ import print_function

File: examples/pidof.py [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    return pids
+
+
+def main():
+    if len(sys.argv) != 2:
+        sys.exit('usage: %s pgm_name' % __file__)
+    else:
+        pgmname = sys.argv[1]
+    pids = pidsof(pgmname)
+    print " ".join(pids)
```
Comment (giampaolo) [inline examples/pidof.py]: put parentheses areound print()

File: examples/pidof.py [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+"""
+
+import psutil
+import sys
+
+
+def pidsof(pgm):
+    pids = []
+    for proc in psutil.process_iter():
+        process = proc.as_dict()
```
Comment (giampaolo) [inline examples/pidof.py]: there's no need to call as_dict() (which retrieves *all* process info; instead just call proc.cmdline()

Comment (giampaolo) [inline examples/pidof.py]: also, put a try/except statement around proc.cmdline() like this:

try:
     cmdline = proc.cmdline()
except psutil.Error:
     continue

File: examples/pidof.py (hunk 1 above)
Comment (karthikrev) [inline examples/pidof.py]: Hi ..Thanks for quick response ..  I am newbie .. Please advice if I should use 
    # Copyright (c) 2009, Giampaolo Rodola' 
or 
   # Copyright (c) 2009, Karthik (my name) ?

Comment (giampaolo) [inline examples/pidof.py]: I guess you could put both names.

**Comment by giampaolo** [comment]
OK, this is now included. Thanks.

**Pull request merged**